runtime/gc_vlhgc/SchedulingDelegate.cpp

```diff
@@ -846,8 +848,24 @@ MM_SchedulingDelegate::measureConsumptionForPartialGC(MM_EnvironmentVLHGC *env,
 		/* this must be the first PGC after a GMP. Since the GMP affected reclaimable memory, we have no reliable way to measure consumption for this cycle */ 
 		Trc_MM_SchedulingDelegate_measureConsumptionForPartialGC_noPreviousData(env->getLanguageVMThread());
 	} else {
-		/* Use a signed number. The PGC may have negative consumption if it recovered more than an Eden-worth of memory, or if the estimates are a bit off */
-		IDATA regionsConsumed = (IDATA)_previousReclaimableRegions - (IDATA)currentReclaimableRegions;
+        MM_CopyForwardStats *copyForwardStats = &static_cast<MM_CycleStateVLHGC*>(env->_cycleState)->_vlhgcIncrementStats._copyForwardStats;
+        MM_CompactVLHGCStats *compactStats = &static_cast<MM_CycleStateVLHGC*>(env->_cycleState)->_vlhgcIncrementStats._compactStats;
+
+        /* TODO: try to remove _previousReclaimableRegions/currentReclaimableRegions, as it does not seem to be used anymore,
+         * and it was not quite correct, since it was not accounting for heap resizing changes (while it should)
+         * and also accounting for sweep reclamation (while it should not)
+         */
```

Comment:
Should indentation for a few lines above be fixed by shifting right?

Reply:
I think I fixed it (copy-paste problems from VI to IDE).